docs/framework/unmanaged-api/profiling/icorprofilerinfo-setilinstrumentedcodemap-method.md

```diff
@@ -79,6 +79,8 @@ The debugger will assume that each old offset refers to an MSIL offset within th
 
   - A new offset of 20 or higher will be mapped to old offset 9.
 
+Before .NET Framework 4.0, `rgILMapEntries` array should have been allocated using the [CoTaskMemAlloc](/windows/desktop/api/combaseapi/nf-combaseapi-cotaskmemalloc) method and the profiler shouldn't have attempt to free this memory.
```

Comment:
It may be better to say `In the .NET Framework version 2.0, ...` to match the style used in the rest of the profiler docs.